src/view/commands.rs

```diff
@@ -27,11 +27,13 @@ pub trait ViewCommand {
 pub enum ViewCommandChoice {
     SubmitRequest {
         request: RequestData,
+        print_body_only: bool,
```

Comment:
Using the "ViewOption" of last comment you can replace this field by the struct created.